lib/sms.js

```diff
@@ -189,6 +189,66 @@ class SMS {
     return this._send(opts, false, true)
   }
 
+  sendToMaskedNumber (params) {
```

Comment:
@nicholas-ngatia Why not keep the existing `send()` function and play on the passed params?